lib/getStatistics.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+var Cesium = require('cesium');
+var defaultValue = Cesium.defaultValue;
+var DeveloperError = Cesium.DeveloperError;
+var isDataUri = require('./isDataUri');
+var PrimitiveHelpers = require('./PrimitiveHelpers');
+
+var defined = Cesium.defined;
+
+module.exports = getStatistics;
+
+/**
+ * Creates an object type for glTF Statistics.
+ *
+ * @property {String} gltfPath The path to the glTF file.
+ * @property {Number} buffersSizeInBytes The total size of all the buffers.
+ * @property {Number} numberOfImages The number of images in asset.
+ * @property {Number} numberOfExternalRequests The number of external requests required to fetch the asset data.
+ * @property {Number} numberOfDrawCalls The number of draw calls required to render the asset.
+ * @property {Number} numberOfRenderedPrimitives The total number of rendered primitives in the asset.
```

Comment:
Add `(e.g. triangles)` to the end of the description (mimics the toString function).